src/systems/audio-gain-system.js

```diff
@@ -1,16 +1,15 @@
-import { SourceType } from "../components/audio-params";
-
 const CLIPPING_GAIN = 0.0001;
 
 export class GainSystem {
   constructor() {
     this.sources = [];
-    window.APP.store.addEventListener("statechanged", this.updatePrefs.bind(this));
+    this.onPrefsUpdated = this.updatePrefs.bind(this);
```

Comment:
Yep, it would be nice to have a Hubs style guide for all this things that we do in different ways for consistency or at least a list of the main ones.